source/git-promise.js

```diff
@@ -1,8 +1,8 @@
 const child_process = require('child_process');
 const gitParser = require('./git-parser');
 const path = require('path');
-const config = require('./config');
 const winston = require('winston');
+const config = require('./config');
```

Comment:
If you are ordering the imports, might as well order all of them. 😄

Reply:
I have reverted it already but not pushed yet.